src/PathTracedVolume.ts

```diff
@@ -492,11 +492,11 @@ export default class PathTracedVolume implements VolumeRenderImpl {
       // TODO expand to 16-bpp raw intensities?
       this.pathTracingUniforms.gIntensityMax.value.setComponent(
         i,
-        this.volume.channels[channel].histogram.getMax() / 255.0
+        this.volume.channels[channel].histogram.getDataMax() / 255.0
       );
       this.pathTracingUniforms.gIntensityMin.value.setComponent(
         i,
-        this.volume.channels[channel].histogram.getMin() / 255.0
+        this.volume.channels[channel].histogram.getDataMin() / 255.0
       );
     }
```

Comment:
I made a change since this seemed to be relying on actual intensity values, but I did notice that `gIntensityMax` is totally unused in the shaders. @toloudis should this be removed or out of scope?

Reply:
The change above here seems like it would alter the rendering results for pathtrace. It's being divided by 255 which makes me think it's expected to be in the 0-255 range...  Did you test the before/after?

Reply:
The field is only used in two commented-out lines in `pathtrace.frag`...

Reply:
if gIntensityMin and gIntensityMax are totally unused in path trace, then I wouldn't bother to modify this code at all either.